### ISSUE TITLE
Provide information concerning HashiCorp Vault in CRD

### ISSUE THREAD

**Issue by tomkerkhove** [kedacore/keda#1826]
### Proposal

Provide information concerning HashiCorp Vault in CRD since it's not surfaced yet when doing `k get triggerauthentication`.

### Anything else?

This would be more consistent since other stores are shown.

**Comment by Shubham82** [comment]
Hi @tomkerkhove
Just for confirmation in this, you want that when we run `k get triggerauthentication` then it would show  HashiCorp Vault information also. if yes then in HashiCorp Vault there are the following fields:
   ```
 address
 authentication
 role
 mount
 credential
      token
      serviceAccount
```
which fields you want to show them in the `kubectl get ta` command.

**Comment by tomkerkhove** [comment]
I'd keep it simple for now and just show the address, what do you think?

**Comment by Shubham82** [comment]
yha, I am also thinking of the address field as it is more suited than other fields.

**Comment by Shubham82** [comment]
Hi @tomkerkhove @zroubalik 

Please suggest the name from the following for the HashiCorp Vault field which shows on running `kubectl get ta`
```
1) HashiCorpVault-ADDRESS
2) HashiCorp Vault-Address
3) Hashicorp-Address
4) HashiCorp-ADDRESS
5) HASHICORP-ADDRESS
```
Any other name you want to add other than this.

**Comment by zroubalik** [comment]
`HCVaultAddress` or `VaultAddress` or? I keep it on @tomkerkhove and his preference.

**Comment by tomkerkhove** [comment]
Let's go with `VaultAddress`

**Comment by Shubham82** [comment]
Thanks, @tomkerkhove @zroubalik 
`VaultAddress` seems good.

**Comment by Shubham82** [comment]
fixed for both ta and cta  here #1862
